.circleci/config.yml

```
@@ -551,6 +551,24 @@ jobs:
           environment:
             VARIANT: '_partial'
             SUBDIR: 'fmriprep-partial'
+      - run:
+          name: Re-run fMRIPrep on single run of task data, without T1w in BIDS
+          no_output_timeout: 2h
+          command: |
+            mkdir -p /tmp/${DATASET}/work_partial_noT1w
+            chmod a+rwX /tmp/${DATASET}/work_partial_noT1w
+            rm -Rf /tmp/data/${DATASET}/sub-01/anat
+            FASTRACK_ARG="--derivatives anat=/tmp/${DATASET}/smriprep"
+            fmriprep-docker -i nipreps/fmriprep:latest \
+                -e FMRIPREP_DEV 1 --user $(id -u):$(id -g) \
+                --network none \
+                --config $PWD/nipype.cfg -w /tmp/${DATASET}/work_partial_noT1w \
+                /tmp/data/${DATASET} /tmp/${DATASET}/fmriprep-partial-noT1w participant \
+                --fs-subjects-dir /tmp/${DATASET}/freesurfer \
+                ${FASTRACK_ARG} \
+                --sloppy --write-graph --mem-mb 14336 \
+                --output-spaces MNI152NLin2009cAsym fsaverage5 \
+                --nthreads 4 --cifti-output --project-goodvoxels -vv
```

This reuses the latest working directory and runs minimal. I think this should take very little time while still hitting the new code.

```suggestion
            rm -Rf /tmp/data/${DATASET}/sub-01/anat
            FASTRACK_ARG="--derivatives anat=/tmp/${DATASET}/smriprep"
            fmriprep-docker -i nipreps/fmriprep:latest \
                -e FMRIPREP_DEV 1 --user $(id -u):$(id -g) \
                --network none \
                --config $PWD/nipype.cfg -w /tmp/${DATASET}/work_partial \
                /tmp/data/${DATASET} /tmp/${DATASET}/fmriprep-partial-noT1w participant \
                --fs-subjects-dir /tmp/${DATASET}/freesurfer \
                ${FASTRACK_ARG} \
                --sloppy --write-graph --mem-mb 14336 \
                --level minimal --nthreads 4 -vv
```